src/Symfony/Component/Notifier/Bridge/Esendex/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+* [BC BREAK] Changed signature of `EsendexTransport::__construct()` method from:
```

Comment:
Shouldn't it be reported in `UPGRADE-5.3.md`

Reply:
Not sure 🧐